tests/flow/includes.py

```diff
@@ -10,12 +10,11 @@
 from skimage.io import imread
 from skimage.transform import resize
 
-try:
-    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../../deps/readies"))
-    import paella
-except:
-    pass
 
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../../opt/readies"))
+import paella
+
+ROOT = os.environ.get("ROOT")
```

Comment:
ROOT = os.environ.get("ROOT", None)
if ROOT is None:
    sys.stderr.write("ROOT was not defined in the environment.\n")
    sys.exit(1)